src/components/SearchForm/SearchForm.js

```diff
@@ -11,11 +19,12 @@ function Search(props) {
           type="text"
           className="search__input"
           placeholder="Enter topic"
+          onChange={(e) => setKeyWord(e.target.value)}
         ></input>
         <button
           type="submit"
           className="search__button"
-          onClick={props.onSearchClick}
+          onClick={handleSearchClick}
```

Comment:
` onClick={handleSearchClick}`

please, delete it from the button

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses